### PR TITLE
Add Cursed 2020

### PR DESCRIPTION
**Pull request by will-chow**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -804,6 +804,12 @@ The natural choice for those who read the Witcher book series, The Witcher tv ad
 
 > Geralt of Rivia, a solitary monster hunter, struggles to find his place in a world where people often prove more wicked than beasts.
 
+#### [Cursed](https://www.imdb.com/title/tt8210856/?ref_=fn_al_tt_1) (2020-?) [7]
+
+Reimagining of the Lady in the Lake with young King Arthur and Merlin by the legendary [Frank Miller](https://en.wikipedia.org/wiki/Frank_Miller_(comics)).  [@will-chow](https://github.com/will-chow)
```
Comment (RichardLitt) [inline README.md:809]: Why is it awesome?

**Review by RichardLitt — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (will-chow) [reply README.md:809]: This rendition of Merlin was awesome, very different take from previous versions.  The mythical world has some twists and spin on favorites like Lord of the Rings and Witcher.

Comment (RichardLitt) [reply README.md:809]: Cool. Could you add that to this description?

Comment (will-chow) [reply README.md]: Added.  Thanks!

**Comment by RichardLitt** [comment]
Thanks!

File: README.md (hunk 1 above)
Comment (will-chow) [reply README.md]: Sorry, I placed my review on the bottom instead of alphabetically.  Thanks for reordering!

Comment (RichardLitt) [reply README.md]: It happens. No worries.

**Pull request merged**